modules/packages/LinearAlgebra.chpl

```diff
@@ -702,6 +702,30 @@ proc _matmatMult(A: [?Adom] ?eltType, B: [?Bdom] eltType)
   return C;
 }
 
+/*
+  Returns the inverse of ``A`` square matrix A.
+  
+  
+    .. note::
+
+      This procedure depends on the :mod:`LAPACK` module, and will generate a
+      compiler error if ``lapackImpl`` is ``none``.
+*/
+proc inv (in A: [?Adom] ?eltType) where usingLAPACK {
```

Comment:
Is the reason for `in` intent to create a copy of A and not overwrite the data in it?

If so, I think we should make the intent `const ref` and create a copy in the implementation explicitly.  Then, control this behavior with an optional flag like `param overwrite=false` or something. If the array is large the use may want to discard it and calculate the inverse in-place. That is [what scipy does](https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.inv.html#scipy.linalg.inv)